root/etc/e-smith/templates/etc/sudoers.d/30nethserver_cockpit_roles/10base

```diff
@@ -0,0 +1,99 @@
+{
+
+    use strict;
+    use warnings;
+    use JSON;
+    #use esmith::ConfigDB;
+
+
+    sub read_json
+    {
+        my $file = shift || return undef;
+        return undef if (! -e $file);
+
+        my $json;
+        {
+            local $/; #Enable 'slurp' mode
+            open my $fh, "<", $file or return undef;
+            $json = <$fh>;
+            close $fh;
+        }
+
+        return decode_json($json);
+    }
+
+
+
+    my $jsonFile = '/etc/nethserver/cockpit/authorization/roles.json';
+    my $data = read_json ($jsonFile); 
+
+    foreach my $key (keys %{$data}) {
```

Comment:
I would love to see a map instead a set of nested if.

A map where you have the view as the key, and the controllers as values.

Something like:
```perl
%controllers = (
  "users" => ("system-roles", "system-password-policy"),
   ...
);
```